salesforce-cdp/1.2/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,7 @@
+.xref:index.adoc[Salesforce Data Cloud Connector]
+* xref:index.adoc[Salesforce Data Cloud Connector Overview]
+* xref:salesforce-cdp-connector-studio.adoc[Using Anypoint Studio to Configure Salesforce Data Cloud Connector]
```

Comment:
Have we begun adding references to using ACB as well or not yet? The only IDE I see referenced in these docs is Studio.

Reply:
@kckane-mulesoft  Only Studio for this connector right now.

Reply:
We have begun. But not for this connector.